build/sdk_update.go

```diff
@@ -0,0 +1,320 @@
+package build
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"text/template"
+	"unicode"
+	"unicode/utf8"
+
+	"gopkg.in/bblfsh/sdk.v2/assets/skeleton"
+	"gopkg.in/bblfsh/sdk.v2/driver/manifest"
+)
+
+const (
+	tplExt      = ".tpl"
+	manifestTpl = manifest.Filename + tplExt
+)
+
+func genEnvBool(key string) bool {
+	str := os.Getenv(key)
+	if str == "" {
+		return false
+	}
+	v, err := strconv.ParseBool(str)
+	if err != nil {
+		panic(err)
```

Comment:
Hmm. I thought about proposing this solution, and didn't on the grounds that optional settings in the environment shouldn't be able to crash the program. I think it's defensible, but would suggest a more detailed error message:

```go
log.Panicf("Invalid value %q for environment flag %q: %v", v, key, err)
```

or words to that effect. Because, importantly, if this _does_ happen, someone is going to have to figure out what to fix in their config files.